### PR TITLE
Creates "AI Ethics Speaker Series" page, refactors several CSS utilities

### PR DESCRIPTION
**Pull request by a17ung**
~I had trouble adding a new CSS file specific for the page so I ended up using a combo of HTML styles and _citylab.scss (def want to change later). I also created a new page for the speaker series because I thought it'd be nice to have a directory for when they complete more events! Said page is definitely not done but I'm going to work on it in the next few days or so. Still not sure about the layout/amount of info on the page as well. I also added a picture of myself on the team page :D~

see matt's comment below

### CODE REVIEW

File: _team/aung.md.orig [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+lastname: "Ung"
+group: "member"
+graduating_year: 2023
+github: "a17ung"
+title: "Emerson Instructor"
+secondary: "Developer"
+pronouns: "she/her"
+img: "aung.jpg"
+
+---
```
Comment (malsf21) [inline _team/aung.md.orig:12]: Is this file an artefact from the merge conflict? I don't think we should merge it into `master`

File: events/fair-ml.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		<h2 class = "text-2x"> Why does fair machine learning matter?
+		</h2>
+		<p style = "font-size: 115%;">
+			Have you ever wondered... Can AI discriminate? Of course! Thanks to Catherine Yeo, we were inspired to teach others about fair machine learning. Check out her work below to learn more.
+		</p>
+
+		<ul style = "font-size: 150%;">
+			<li> <a class="tla-link" href="http://catherineyeo.tech/ai_fairness_course" target="_blank" rel = "noopener noreferrer">Fairness in AI and Algorithms Class Syllabus </a>
+			</li>
+			<li> <a class = "tla-link" href="https://slack-files.com/files-pri-safe/TA3824MMX-F01BCVAJZ6X/fairness-ethics-in-ai-talk.pdf?c=1607066003-e8f6a1cc96e00768" target = "_blank" rel = "noopener noreferrer"> Catherine's Slides </a>
```
Comment (malsf21) [inline events/fair-ml.html:30]: I wouldn't personally use a Slack link to share a file - let's instead 

* link to the slides on Catherine's website (if they're there, this is the preferred option - we should give credit)
* make a copy and put it in the site itself 
* share a google drive link

File: events/fair-ml.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+<hr class="divider" />
+
+<div class="citylab-section" style = "margin-top: 2em;">
+	<div class = "grid-col">
+		<img class = "img-responsive" src = "{{site.baseurl}}/img/undraw/content.svg" alt = "drawing related to content" />
+	</div>
+	<div>
+		<h2 class = "text-2x"> Why does fair machine learning matter?
+		</h2>
+		<p style = "font-size: 115%;">
```
Comment (malsf21) [inline events/fair-ml.html:23]: In general, I would avoid using inline style tags when possible - we should define this as a class somewhere (maybe in `_basics.scss`)

File: events/fair-ml.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+			Have you ever wondered... Can AI discriminate? Of course! Thanks to Catherine Yeo, we were inspired to teach others about fair machine learning. Check out her work below to learn more.
+		</p>
+
+		<ul style = "font-size: 150%;">
```
Comment (malsf21) [inline events/fair-ml.html:27]: We've already defined a class that does this (or something similar) - `.text-15x`!

File: events/fair-ml.html [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+title: "Fair ML Speaker Series"
+permalink: "/fair-ml"
+---
+
+<h1 class="title page-title"> Fair ML Speaker Series</h1>
+
+<p style = "font-size: 115%;">
+	Thanks to our <a class="tla-link" href="{{site.baseurl}}/speaker-series"> Speaker Series Team </a>, our Fair ML Speaker Series was a success! <b> Learn all about it on this page. </b>
+
+</p>
```
Comment (malsf21) [inline events/fair-ml.html:12]: I think this content should instead generally explain what the Fair ML speaker series is. You don't necessarily have to write the copy - feel free to reach out to someone in the slack channel for help on this!

File: events/fair-ml.html [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+<hr class="divider" />
+
+<div class="citylab-section" style = "margin-top: 2em;">
```
Comment (malsf21) [inline events/fair-ml.html:16]: I would maybe swap the order of this section and the predictive policing one - from a framing perspective, this is a "more resources"/credits type of content block, while the other is the main content. We can also discuss how to revise the copy later!

File: events/fair-ml.html [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+	<div class = "grid-col">
+		<img class = "img-responsive" src = "{{site.baseurl}}/img/undraw/content.svg" alt = "drawing related to content" />
```
Comment (malsf21) [inline events/fair-ml.html:18]: A slightly more descriptive alt tag would be nice!

File: events/fair-ml.html [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+	<p style = "font-size: 115%;">
+		So you know AI can be biased, but how? Predictive policing uses ML and AI to try to predict crime. Often, the data used is skewed by human bias and inconsistent crime reports. Want to learn more about predictive policing? We got you covered!
+	</p>
+
+	<div class = "button-container">
+		<a class="button button-dark" href="
+		https://docs.google.com/presentation/d/1IRvK3awKEll42cs8MTc3YdCONHa4E_3hV-dU2inlGQ8/edit#slide=id.g9fdeae80bb_0_18" target ="_blank" rel = "noopener noreferrer"><span class = "fa fa-link"></span> Link to Case Study</a>
+		
+		<a class="button button-dark" href="
+		https://docs.google.com/document/d/1f1XhxGFxA-JAz2Dw6I5gy5jS2Z_oW3aBKmJUkU5PBqA/edit" target="_blank" rel="noopener noreferrer"><span class = "fa fa-link"> </span> Link to ML Wiki</a>
```
Comment (malsf21) [inline events/fair-ml.html:56]: Nitpicky, but a different Icon here might be nice! Something that feels more "wiki-ish"

File: events/fair-ml.html [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+	<h2 class="title text-2x">
+		Predictive Policing
+	</h2>
+	<p style = "font-size: 115%;">
+		So you know AI can be biased, but how? Predictive policing uses ML and AI to try to predict crime. Often, the data used is skewed by human bias and inconsistent crime reports. Want to learn more about predictive policing? We got you covered!
+	</p>
+
+	<div class = "button-container">
+		<a class="button button-dark" href="
+		https://docs.google.com/presentation/d/1IRvK3awKEll42cs8MTc3YdCONHa4E_3hV-dU2inlGQ8/edit#slide=id.g9fdeae80bb_0_18" target ="_blank" rel = "noopener noreferrer"><span class = "fa fa-link"></span> Link to Case Study</a>
```
Comment (malsf21) [inline events/fair-ml.html:53]: Nitpicky, but a different Icon here might be nice! Something that feels more "presentation"/"case-study"-ish. You can find the list of free font awesome icons at [the FA5 website](https://fontawesome.com/icons?d=gallery)

Comment (malsf21) [inline events/fair-ml.html:53]: Minor formatting, but is there a reason that this line break is here?

File: _team/aung.md [hunk 10]
```diff
@@ -2,10 +2,11 @@
 firstname: "Amanda"
 lastname: "Ung"
 group: "member"
+graduating_year: 2023
+github: "a17ung"
 title: "Emerson Instructor"
 secondary: "Developer"
 pronouns: "she/her"
 img: "aung.jpg"
```
Comment (malsf21) [inline _team/aung.md:10]: Think you accidentally deleted the old image and kept the artifact (`aung.jpg.orig`) - you may want to revert that so your team page works!

File: speaker-series.html [hunk 11]
```diff
@@ -0,0 +1,7 @@
+---
+layout: default
+title: "Speaker Series"
+permalink: "/speaker-series"
+---
+
+<h1 class="title page-title"> Speaker Series </h1>
```
Comment (malsf21) [inline speaker-series.html:7]: For now we can delete this page, maybe we'll revisit it another time!

**Review by malsf21 — CHANGES_REQUESTED** [review]
Hey Amanda,

Thanks for your work on this so far, the page is looking great! Left a few points of feedback; here's the general gist:

* I think you may have accidentally messed up a few files on the merge that make your team page entry not work properly! 
* In terms of copy and content, you may want to revise how we're framing it with the team - feel free to send the draft preview of the PR to the channel. I think Arjun may have some specific asks in particular
* Left some minor comments on reusable CSS (when possible, you should avoid the style tag unless you have a good reason to use it). 

Also, to answer your question on adding a new .scss file, it's a two-step process!

1. first, add your `_whatever.scss` file in the `_sass` folder
2. next, you'll need to import your new file in `css/main.scss` - as long as the `@import` statement is under variables and basics, everything should work as intended!

This is looking great for a first iteration! If you're able to knock out most of the feedback, I think all that's probably left is some minor copy changes; then, it'll be good to merge!

**Comment by malsf21** [comment]
Going to take over this as Amanda takes a quick break from the dev team!

**Comment by malsf21** [comment]
Just wrapped up Amanda's changes, and adding some things since the ticket was assigned. Here's a complete diff from master:

* created a new "AI Ethics Speaker Series" page; links to it from the footer
* writes out AI Ethics content for four categories + thanks to Catherine (hosting her slides, since they're not on her website)
* refactors `_citylab.scss` into `_event.scss` for shared utilities; renames `.citylab-*` classes to `.event-*`
* adds `.border-ai` utility class
* **importantly, applies `*{ box-sizing: border-box;}`. this has unintended side effects to the rest of the site**
* updates Amanda's profile card

**Comment by malsf21** [comment]
Because this change has quite a few side effects, going to ask for a double-review on these changes!

**Review by lishamohan — COMMENTED** [review]
Looks good to me! Only notable differences I noticed from changing the box-sizing is that the "join us" button got a lil squashed and the pictures on the team page are smaller. Both can be fixed real quick; lmk if you want me to make a quick commit to fix them (i would just suggest the changes on this PR but i don't think i can since those files weren't edited in this PR)

**Comment by malsf21** [comment]
> Only notable differences I noticed from changing the box-sizing is that the navbar "join us" button got a lil squashed and the pictures on the team page are smaller.

Great catch, I noticed the navbar button but not the team page issues! Feel free to add on to this branch if you want 😊  (but also I can do it if you're busy)!

**Comment by lishamohan** [comment]
Okay there we go I think that should do it!

**Comment by malsf21** [comment]
> Okay there we go I think that should do it!

Can verify that your changes work as intended, thanks for the quick fix! Just waiting on Leo's feedback then 😊

**Comment by malsf21** [comment]
Pinging @krashanoff again on this for a second look!

**Review by krashanoff — APPROVED** [review]
Quite a many changes, but after looking through the styles, it looks good to me. Refactors make sense and the page looks good. Thanks for adding this all Amanda, thanks for the fix after the `border-box` change Lisha, and thanks for the thorough review Matt!

**Pull request merged**